src/domain/course-catalog/application/use-cases/get-user-overall-progress.use-case.spec.ts

```
@@ -0,0 +1,494 @@
+      await lessonProgressRepository.save(progress3);
+
+      const result = await useCase.execute({ userId });
+
+      expect(result.isRight()).toBe(true);
+      if (result.isRight()) {
+        const response = result.value;
+        
+        expect(response.tracksProgress.mostAdvancedTrack).toBeDefined();
+        if (response.tracksProgress.mostAdvancedTrack) {
+          expect(response.tracksProgress.mostAdvancedTrack.id).toBe(trackId1);
+          expect(response.tracksProgress.mostAdvancedTrack.progress).toBeGreaterThan(0);
```

ESLint

[error] Replace `response.tracksProgress.mostAdvancedTrack.progress` with `⏎············response.tracksProgress.mostAdvancedTrack.progress,⏎··········`

(prettier/prettier)